pkg/kdump/kdump-view.jsx

```diff
@@ -414,13 +414,27 @@ export class KdumpPage extends React.Component {
 # A reboot will be required if crashkernel was not set before
 kdumpctl reset-crashkernel`;
         }
-        const shell = `
+        let shell;
+        if (this.state.os_release.NAME?.includes('MicroOS')) {
+            enableCrashKernel = `
+# A reboot will be required if crashkernel was not set before
+transactional-update setup-kdump`;
+            shell = `
+cat > /etc/kdump.conf << EOF
+ ${kdumpconf}
+EOF
+${enableCrashKernel}
+# run systemctl enable --now kdump.service after rebooting
```

Comment:
How is that different to `systemctl enable --now kdump.service` that runs on other OSes, except for being manual? Or if that can't start yet, could it at least be `enable` without `--now`, so that it auto-starts after boot?

In other words, is there a way to not make this completely special?

Reply:
In the case kdump isn't installed, the service will not exist until you run setup-kdump which will pull in the package.

Reply:
Sure, but setup-kdump is called above already as part of `enableCrashKernel`?

Reply:
Microos is transactional. The service will only appear after a reboot

Reply:
I was mistaken some part of `setup-kdump`. it also enables the service, so you just need to reboot after running it